### DISCUSSION TITLE
Images won't index

### DISCUSSION THREAD

**Discussion by hamilTek** [photoprism/photoprism#4569] [Q&A]
Windows 11
WSL 2
I've finished all steps in the Docker Compose walkthrough for windows and for the life of me I don't understand what's happening (though humble enough to assume I'm making a bonehead mistake).

My docker-compose.yml
![image](https://github.com/user-attachments/assets/44bab0ac-ed02-40a6-9ef4-eeaf7bae75f3)

My L: drive is a mounted network drive (NAS) that I can navigate to and view pictures from
My P: drive is a mounted ssd 

When I perform an index photoprism picks up absolutely nothing.  My 'Shared Pictures' folder has folders by year, then month, but photoprism simply will not traverse the tree and I don't now why.  The logs show that it did exactly what it did, but no errors.

I've tried - `docker compose exec photoprism photoprism reset` to remove the database and start fresh
I've tried - `docker compose exec photoprism photoprism index -f` as well as from the front-end but it just won't pick up the whole drive.

I welcome who ever wants to tell me where I failed to read the basic instruction (as I'm sure that's what it is) but I just haven't found it.

**Comment by graciousgrey** [comment]
You can find our troubleshooting checklist for missing pictures here:
- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

In case no files can be indexed I would recommend to check the permissions first.

Turning on trace logs and watching the docker logs while indexing might give you further insights:

- https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3
